eng/pipelines/aggregate-reports.yml

```diff
@@ -126,6 +123,30 @@ jobs:
         # GdnBreakOutputBaseline: baseline
       condition: succeededOrFailed()
       continueOnError: true
+    - pwsh: |
+        $results = (Get-Content $(Pipeline.Workspace)\.gdn\.r\CredScan\001\CredScan-matches.sarif | ConvertFrom-Json).runs.results
```

Comment:
Is this consistent for all languages? If so we may want to consider sharing this script.

Reply:
Right, it is consistent for all languages.